Add option to disable versioning in EOS upon PUT request (#4864)
cs3org/reva: changelog/unreleased/disable-versioning.md

Enhancement: Add HTTP header to disable versioning on EOS

> This enhancement introduces a new header, `X-Disable-Versioning`, on PUT requests. Whenever this header is set, with any value (but `true` is recommended), EOS will not version this file save.

Reviewer: Is it really any value? Isn't any value convertible by `ParseBool`? I would rephrase for something like

```suggestion
This enhancement introduces a new header, `X-Disable-Versioning`, on PUT requests. EOS will not version this file save whenever this header is set with a truthy value.
```
